app/models/vm_or_template/operations.rb

```diff
@@ -112,11 +112,10 @@ def validate_vm_control
             end
       unsupported_reason_add(:control, msg) if msg
     end
-  end
 
-  def validate_terminate
-    return {:available => false, :message => 'The VM is terminated'} if self.terminated?
-    {:available => true, :message => nil}
+    supports :terminate do
```

Comment:
is this the only place in the code?
What about `templates` ? Can they be terminated?